src/rpc-util.js

```diff
@@ -22,7 +22,7 @@ export function createRpcService({ name, routes }) {
             result = methodResolver(params);
           }
 
-          return Promise.resolve(result).then((value) => {
+          return Promise.resolve(result).then(value => {
```

Comment:
should the promise have an error handler?

Reply:
In this case we are just using a logger for debug purposes. Errors will be handle by rpc client, since the service will just throw it.